collectors/proc.plugin/proc_net_dev.c

```diff
@@ -826,6 +834,14 @@ int do_proc_net_dev(int update_every, usec_t dt) {
         if(unlikely(!d->enabled))
             continue;
 
+        // See https://github.com/netdata/netdata/issues/15206
+        // This is necessary to prevent the creation of charts for virtual interfaces that will later be 
+        // recreated as container interfaces (create container) or
+        // rediscovered and recreated only to be deleted almost immediately (stop/remove container)
+        if (d->virtual && (now - d->discover_time < NETDEV_VIRTUAL_COLLECT_DELAY)) {
+            continue;
+        }
```

Comment:
A better solution would be to delay only specific interfaces that are likely to be container interfaces (e.g. `veth*`, `kvm*`, `cri*`), but I don't know all naming patterns. 

Considering that the problem causes [all network interface charts to be kinda broken](https://discord.com/channels/847502280503590932/1118901743732924496/1118940090396332042) when using a relatively large time range (e.g. the last N hours), I decided to just delay them all. The disadvantage of this is obvious - other virtual interfaces (e.g. bridge, link) will have 15-second gaps after each restart. Not that bad if we assume that usually you restart ND once a day (update).